releases/R2024b/README.md

```diff
@@ -0,0 +1,286 @@
+    * Look for the key named `MATLABWebAppServerOIDCRedirectUrl` and copy the corresponding URL listed under value.
+    * Use this URL in the `webapps_authn.json` file.
+
+## Create Self-signed Certificate
+For information on creating a self-signed certificate, see [Create and Sign an X509 Certificate](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/configuring-https-ssl.html).
+
+## Upload Self-signed Certificate to AWS Certificate Manager
+
+1. Open the AWS Certificate Manager.
+2. Click the button at the top of the page to **Import a certificate**.
+3. Copy the contents of the `.crt` file containing the certificate into the field labeled **Certificate body**.
+4. Copy the contents of the `.pem` file containing the private key into the field labeled **Certificate private key**.
+5. Leave the field labeled **Certificate chain** blank.
+6. Click the button labeled **Review and import**.
+7. Review the settings and click the **Import** button.
+8. Copy the value of the Amazon Resource Name (ARN) field from the **Details** section of the certificate.
+
+The ARN value that you copied should be pasted into the **ARN of SSL Certificate** parameter of the template in [Step 2](#step-2-configure-the-stack).
+
+## View Logs
+Logs are available in AWS CloudWatch. 
+1. In the AWS management console, select the stack you deployed. 
+1. In the *Stack details* for your stack, click the **Outputs** tab.
+1. To view logs related to the cloud console and the MATLAB Web App Server workers, look for the key named `MATLABWebAppServerLogGroup`, and click the corresponding URL listed under value.
+
+## Get Network License Manager MAC Address
+1. In the AWS management console, select the stack that you deployed. 
+1. In the *Stack details* for your stack, click the **Outputs** tab.
+1. Look for the key named `MATLABWebAppServerLicenseManager`, and click the corresponding URL listed under value.
+1. Log in to the Network License Manager for MATLAB dashboard using the following credentials:<br>
+Username: **manager**<br>
+Password: Enter the password you provided while creating the stack.
+1. Click **Administration** > **Manage License**.
+1. Copy the license server MAC address displayed at the top.
+
+When you deploy the MATLAB Web App Server reference architecture, a network license manager is automatically deployed. You can also use an existing license manager that is located in the same VPC and the security group of the MATLAB Web App Server EC2 instance. For detials, see [Network License Manager for MATLAB](https://github.com/mathworks-ref-arch/license-manager-for-matlab-on-aws).
+
+>**NOTE**: The license manager MAC address is available only after the stack creation is complete.
+
+## Delete Your Stack
+To delete the stack:
+1. Log in to the AWS Console.
+3. Go to the CloudFormation page and select the stack you created.
+3. Click **Delete**.
+
+# FAQ
+
+## How do I use an existing VPC to deploy MATLAB Web App Server?
```

Comment:
I don't think this section is needed. You already have the button/link to deploy to an existing VPC towards the top of the Readme.